0.1.Copy data from Excel codes.R

```diff
@@ -1,79 +1,75 @@
-rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
-gc() #free up memrory and report the memory usage.
+#rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
+#gc() #free up memrory and report the memory usage.

```

Comment:
If you are running this either via `CTRL`-`SHIFT`-`K` or by sourcing the script in a new RStudio session you just opened in a newly cloned directory, you won't need to free up memory.